src/components/Tooltip/TooltipSense.js

```diff
@@ -0,0 +1,26 @@
+import _ from 'underscore';
+import CONST from '../../CONST';
+
+let active = false;
+const debouncedDeactivate = _.debounce(() => {
+    active = false;
+}, CONST.TIMING.TOOLTIP_SENSE);
+
+function activate() {
+    active = true;
+    debouncedDeactivate.cancel();
+}
+
+function isActive() {
+    return active === true;
+}
+
+function deactivate() {
+    return debouncedDeactivate();
+}
```

Comment:
NAB: A bit nitty-gritty, but I would keep the order alphabetical same as you have it in the export default on lines 23-25.